data/harisdev.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "Haris Khan",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Computer Science guy in a Software Engineering world!",
+    "avatar": "https://avatars.githubusercontent.com/u/81189198?v=4.png"
```

Comment:
```suggestion
    "avatar": "https://avatars.githubusercontent.com/u/81189198?v=4.png",
```

Reply:
As mentioned by Amanda, the avatar is no longer required.